docs/source/tutorial/checkpoints.rst

```diff
@@ -140,6 +140,118 @@ regular checkpoints as defined above, e.g. with this code:
 
 Note: Use this argument with caution, since every failed training loop will create a distinct checkpoint file.
 
+.. _byod_and_checkpoints_training:
+
+Checkpoints and Bring Your Own Data - Resuming training
+-------------------------------------------------------
+When continuing the training or general usage of a model it is of vital importance that the ``entity_to_id`` and
+``relation_to_id`` mappings that were used when saving the checkpoint are the same as when continuing to use the model.
+When using datasets provided by PyKEEN we have you covered, since PyKEEN makes sure this is the case. However,
+when using your own datasets PyKEEN is dependent on you, the user, to make sure this is the case. But don't worry,
+PyKEEN got you covered and will help you to make sure everything is right. To see how, let's look at a typical case
+of using your own data:
+
+>>> from pykeen.pipeline import pipeline
+>>> from pykeen.triples import TriplesFactory
+>>> train = TriplesFactory.from_path('/my/data/train_triples_1.txt')
```

Comment:
It would be better if this were code that would actually run. Could you import from `pykeen.datasets.nations import NATIONS_TRAIN_PATH, ...` and use those?

Reply:
Sure. I was more thinking about making it clear that custom files are used, but I agree that making it executable out of the box is a nice thing to have